src/parquet_impl.cpp

```diff
@@ -1265,9 +1276,16 @@ parquetGetForeignPaths(PlannerInfo *root,
                                  NULL);
 
         /* Create PathKey for the attribute from "sorted" option */
-        attr_pathkeys = build_expression_pathkey(root, (Expr *) var, NULL,
-                                                sort_op, baserel->relids,
-                                                true);
+        // pg16- pdate
+        #if PG_VERSION_NUM > 160000
+            attr_pathkeys = build_expression_pathkey(root, (Expr *) var,
+                                                    sort_op, baserel->relids,
+                                                    true);
+        #else
+            attr_pathkeys = build_expression_pathkey(root, (Expr *) var, NULL,
+                                                    sort_op, baserel->relids,
+                                                    true);
+        #endif
```

Comment:
Usually directives are not indented and the condition should be `PG_VERSION_NUM >= 160000`:
```suggestion
#if PG_VERSION_NUM >= 160000
        attr_pathkeys = build_expression_pathkey(root, (Expr *) var,
                                                sort_op, baserel->relids,
                                                true);
#else
        attr_pathkeys = build_expression_pathkey(root, (Expr *) var, NULL,
                                                sort_op, baserel->relids,
                                                true);
#endif
```